include/api.php

```diff
@@ -367,7 +394,12 @@ function pll_set_term_language( $id, $lang ) {
  * @phpstan-return array<non-empty-string, positive-int>
```

Comment:
I'm not familiar with the PHPStan doc, but it can be empty right?

Reply:
The array can, if it couldn't it would be `non-empty-array<non-empty-string, positive-int>`.
In other words, `non-empty-string` refers to the array key (if there is one).